cdk/lib/cdk-playground.ts

```diff
@@ -24,6 +24,9 @@ export class CdkPlayground extends GuStack {
 
 		const ec2App = 'cdk-playground';
 		const ec2AppDomainName = 'cdk-playground.gutools.co.uk';
+    //New domain added for Fastly disaster recovery testing. Will be deleted after this testing is complete
+    //todo - delete fastlyTestDomainName once the Fastly disaster recovery testing is complete.
+    const fastlyDRTestDomainName = 'cdk-status-resources.guim.co.uk';
```

Comment:
I know we're only testing so it doesn't have a big impact, but I think `guardianapis` would make more sense than `guim` in the context of this app (it's not serving images, but it is serving simple JSON at https://cdk-playground.gutools.co.uk/).

Reply:
good point

Reply:
ok that should be corrected in the latest commit.